binaries/auto_build_setup.py

```diff
@@ -0,0 +1,54 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# NOTE: This file is intended to be used only by the build script
+# and not to be run from the terminal.
+import sys
+import ast
+import setuptools # type: ignore
+
+import streamer_binaries
+
+# The last argument passed MUST be a list of binary file names
+# that we wish include in the build.
+# Parse this argument into a list.
+# This argument will look something like this:
+# 	['binary_1', 'binary_2', 'binary_3', 'binary_4']
+platform_binaries = ast.literal_eval(sys.argv[-1])
```

Comment:
I like this latest change overall, but it seems hacky to encode a python list into a single arg, then parse it using "ast", when we can naturally pass multiple args.  I would prefer we use a more typical command-line convention like "--" to separate the setuptools args from the list of binaries.

For example, the caller would look like:

```py
args = [
  'foo',
  'bar',
  '--py3',
  '--',
] + binary_paths
```

And here we would do something like:

```py
separator_index = sys.argv.index('--')
platform_binaries = sys.argv[separator_index+1:]
sys.argv = sys.argv[:separator_index]
```

What do you think?